packages/components/atoms/f-form-field/src/components/FormDropdown.vue

```diff
@@ -1,13 +1,21 @@
 <template>
     <div
-        :class="$style['c-formDropdown']"
+        :class="[
+            $style['c-formDropdown'],
+            { [$style['c-formField--invalid']]: hasError }
```

Comment:
Looks like the styling is being applied to the `c-formField-field` so you could move this class down onto the select and then change the selector so that it just targets the `c-formField--invalid` class?